src/index.js

```diff
@@ -20,12 +29,74 @@ const positionSeven = board.grid[6];
 const positionEight = board.grid[7];
 const positionNine = board.grid[8];
 
-document.getElementById('one').innerHTML = positionOne;
-document.getElementById('two').innerHTML = positionTwo;
-document.getElementById('three').innerHTML = positionThree;
-document.getElementById('four').innerHTML = positionFour;
-document.getElementById('five').innerHTML = positionFive;
-document.getElementById('six').innerHTML = positionSix;
-document.getElementById('seven').innerHTML = positionSeven;
-document.getElementById('eight').innerHTML = positionEight;
-document.getElementById('nine').innerHTML = positionNine;
+const currentPlayer = () => {
+  whosPlaying.innerHTML = `Player ${game.currentPlayer} moves`
+}
+
+const showBoard = () => {
+  one.innerHTML = positionOne;
+  two.innerHTML = positionTwo;
+  three.innerHTML = positionThree;
+  four.innerHTML = positionFour;
+  five.innerHTML = positionFive;
+  six.innerHTML = positionSix;
+  seven.innerHTML = positionSeven;
+  eight.innerHTML = positionEight;
+  nine.innerHTML = positionNine;
+  currentPlayer()
+}
+
+
+const startGame = () => {
+  showBoard();
+}
+
+startButton.addEventListener("click", startGame)
+
+one.addEventListener("click", function(){
```

Comment:
The anonymous function here can be written in arrow syntax to be more consistent with the rest of your app
ie .
`function() {` on line 56 can be replaced with `() => {`

Reply:
Thanks Dan :)